src/Block/block.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
+// phpcs:disable WordPress.Security.NonceVerification.Recommended
```

Comment:
It'd be great to explain why we disable some of phpcs, maybe like this?
```suggestion
// The URL parameters are received from our OpenID provider and include a nonce inside the JWT token.
// phpcs:disable WordPress.Security.NonceVerification.Recommended
```